docker/transformers-doc-builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8
```

Comment:
Let's try python 3.10 for doc build - could unblock other HF's repositories' workflow

Reply:
confirmed to work with `transformers` and @abhishekkrthakur repository